.github/workflows/continuous-integration.yml

```
@@ -41,25 +41,29 @@ jobs:
 
   test-pip:
     needs: pre-commit
+    continue-on-error: ${{ matrix.experimental || false }}
+    runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         python-version: [ 3.7, 3.8, 3.9, "3.10", "3.11"]
-        markdown-it-py-version: [2.0]
+        markdown-it-py-version: ["2.0"]
         kernel: [true]
         experimental: [false]
         include:
-          - python-version: 3.12
+          - python-version: "3.12"
```

Note that there is/will be `3.13-dev` soon:
https://github.com/actions/setup-python/blob/0d5da6a89a3aa5b4dbd48e38b0c9d678b0caa0e5/docs/advanced-usage.md#using-the-python-version-input

Until it is released, I think this one needs to be:
```suggestion
          - python-version: "3.12.0-alpha - 3.12.0"
```